modules/hal_nordic/nrfx/CMakeLists.txt

```diff
@@ -44,6 +44,8 @@ zephyr_library_sources_ifdef(CONFIG_SOC_NRF9160        ${MDK_DIR}/system_nrf9160
 
 zephyr_library_sources(nrfx_glue.c)
 
+zephyr_compile_definitions_ifdef(CONFIG_LOG_BACKEND_SWO ENABLE_SWO)
+
```

Comment:
Please move this up (to right after the `NRF_TRUSTZONE_NONSECURE` entry), so that all compile definitions are kept together.

Reply:
Thanks for the review, I just did that change.